locales/en-US/app.ftl

```diff
@@ -95,7 +95,11 @@ CallNodeContextMenu--transform-collapse-resource =
 CallNodeContextMenu--transform-collapse-direct-recursion = Collapse direct recursion
     .title =
         Collapsing direct recursion removes calls that repeatedly recurse into
-        the same function.
+        the same function with no intermediate functions on the stack.
```

Comment:
Changes to existing strings require new IDs.
https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html